.github/workflows/nightlies.yml

```
@@ -0,0 +1,52 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+  - cron: '23 1 * * *'
+  # Run on demand with workflow dispatch
+  workflow_dispatch:
+
+jobs:
+  upload_nightly_wheels:
+    name: Upload nightly wheels to Anaconda Cloud
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+
+      # c.f. https://github.com/actions/download-artifact/issues/3#issuecomment-1017141067
+      - name: Download wheel artifacts from last build on 'main'
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PROJECT_REPO=matplotlib/matplotlib
+          BRANCH=main
+          WORKFLOW_NAME="cibuildwheel.yml"
+          ARTIFACT_NAME=wheels
+
+          gh run --repo "${PROJECT_REPO}" list --branch "${BRANCH}" --workflow "${WORKFLOW_NAME}" --json status,databaseId > runs.json
+          # Get latest completed build of wheels
+          RUN_ID=$(cat runs.json | jq -c '[ .[] | select(.status == "completed") ] | sort_by(.databaseId) | reverse | .[0].databaseId')
+          gh run --repo "${PROJECT_REPO}" download "${RUN_ID}" --name "${ARTIFACT_NAME}"
```

I'd also wrap these at 80, though I'm not sure how best to wrap the `jq` script.
```suggestion
          gh run --repo "${PROJECT_REPO}" \
             list --branch "${BRANCH}" \
                  --workflow "${WORKFLOW_NAME}" \
                  --json status,databaseId > runs.json
          # Get latest completed build of wheels
          RUN_ID=$(cat runs.json | \
            jq -c '[ .[] | select(.status == "completed") ] | sort_by(.databaseId) | reverse | .[0].databaseId'
          )
          gh run --repo "${PROJECT_REPO}" \
             download "${RUN_ID}" --name "${ARTIFACT_NAME}"

```

---

> I'd also wrap these at 80

Okay. I'm going to have to rewrite this to deal with the `event` filter so I'll stick to 80 there. 

> though I'm not sure how best to wrap the `jq` script.

Here I personally think sticking to 80 makes the legibility significantly worse. If this is a hard rule then it would be better to split the `jq` steps up into multiple stages.